### PR TITLE
feat: release healthcert

### PR DESCRIPTION
**Pull request by Nebulis**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,3 +51,11 @@
 
 >_If you are uncertain of the structure of an Open Attestation document, you can refer to [this](https://openattestation.com/docs/verifiable-document/raw-document) and append your existing sample document with the required fields.
 ```
+
```
Comment (Nebulis) [inline README.md:54]: check latest todo

**Comment by eujinong123** [comment]
@Nebulis, you created the [notarized healthcert schema](https://github.com/Open-Attestation/schemata/blob/master/src/sg/gov/moh/healthcert/1.0-draft/healthcert-notarised-schema.json). IMO, the very healthcert schema reference should be used instead of openattestation schema reference, in Line No.7.

Btw, `notarisationMetadata.passportNumber` is equal to `fhirBundle.entry[0].identifier[0].value` (patient's passport number)?

File: src/sg/gov/moh/healthcert/1.0/schema.json [hunk 2]
```diff
@@ -299,7 +298,8 @@
         "type": {
           "description": "The kind(s) of organization that this is.",
           "type": "string",
-          "examples": ["Licensed Healthcare Provider", "Accredited Laboratory"]
+          "examples": ["Licensed Healthcare Provider", "Accredited Laboratory"],
+          "enum": ["Licensed Healthcare Provider", "Accredited Laboratory"]
```
Comment (Nebulis) [inline src/sg/gov/moh/healthcert/1.0/schema.json:302]: To revert

**Comment by Nebulis** [comment]
@eujinong123 I don't understand the meaning of the first part of your comment. What is your suggestion?

Regarding the passport number `notarisationMetadata.passportNumber` is supposed to be equal to `fhirBundle.patientEntry.ppnIdentifier.value` where
- patientEntry is the entry with resource for equal to Patient
- ppnIdentifier is the identify with type equal to PPN

The initial information that was forwarded to us mentioned that passport number was not mandatory. That's however different from the latest feedback I received. That aside notarization is meant for more use cases than only healtcert that's why it's there. You don't have to care about this because it only concerns there notarization process

**Comment by eujinong123** [comment]
{
    "$id": "https://schemata.openattestation.com/sg/gov/moh/healthcert/1.0/healtcert-notarised-schema",
    "$schema": "http://json-schema.org/draft-07/schema#",
    "type": "object",
    "allOf": [
         {
               _**"$ref": "https://schema.openattestation.com/2.0/schema.json"**_
         },
         {
              _**"$ref": "https://schemata.openattestation.com/sg/gov/tech/notarise/1.0/notarise-open-attestation-schema.json"**_
         }
     ]
}

As you can see, the `openattestation` schema reference is repeated, isn't it? So my suggestion is to use the `healthcert` schema in the first element of array instead of `openattestation` schema. This file wasn't changed in this PR but I recognized it seems to be wrong recently.

If you want, I will fork and create a PR.

**Comment by Nebulis** [comment]
> {
> "$id": "[schemata.openattestation.com/sg/gov/moh/healthcert/1.0/healtcert-notarised-schema](https://schemata.openattestation.com/sg/gov/moh/healthcert/1.0/healtcert-notarised-schema)",
> "$schema": "http://json-schema.org/draft-07/schema#",
> "type": "object",
> "allOf": [
> {
> _**"$ref": "https://schema.openattestation.com/2.0/schema.json"**_
> },
> {
> _**"$ref": "https://schemata.openattestation.com/sg/gov/tech/notarise/1.0/notarise-open-attestation-schema.json"**_
> }
> ]
> }
> 
> As you can see, the `openattestation` schema reference is repeated, isn't it? So my suggestion is to use the `healthcert` schema in the first element of array instead of `openattestation` schema. This file wasn't changed in this PR but I recognized it seems to be wrong recently.
> 
> If you want, I will fork and create a PR.

Yes you are correct the first element is supposed to be healthcert schema, I made the change

**Pull request merged**